Rubrik/Public/Get-RubrikSettings.ps1

```diff
@@ -0,0 +1,66 @@
+﻿#Requires -Version 3
+function Get-RubrikSettings
+{
+  <#  
+      .SYNOPSIS
+      Connects to Rubrik and retrieves the current Rubrik cluster settings
+            
+      .DESCRIPTION
+      The Get-RubrikSettings cmdlet will retrieve the cluster settings actively running on the system. This does require authentication.
+            
+      .NOTES
+      Adapted by Adam Shuttleworth from scripts by Chris Wahl for community usage
+            
+      .LINK
+      https://github.com/rubrikinc/PowerShell-Module
+            
+      .EXAMPLE
+      Get-RubrikSettings -Server 192.168.1.100
+      This will return the running cluster settings on the Rubrik cluster reachable at the address 192.168.1.100
+  #>
+
+  [CmdletBinding()]
+  Param(
+    # Rubrik server IP or FQDN
+    [Parameter(Mandatory = $true)]    
+    [String]$Server,
```

Comment:
The $Server and $api vars should remain pointed towards global variables that you see in the template.

```
    # Rubrik server IP or FQDN
    [String]$Server = $global:RubrikConnection.server,
    # API version
    [String]$api = $global:RubrikConnection.api
```

Reply:
I updated all functions to include the proper Server and api parameter variables.